src/index.js

```diff
@@ -1,43 +1,136 @@
 
+// Get image for enter icon
 const enterContainer = document.querySelector('.enter-container');
 const enter = new Image();
 enter.src = Enter;
 enterContainer.innerHTML = `<img src=${Enter} class="enter-icon" />`;
 
-const listContainer = document.querySelector('.list-container');
+// Get image for 3 dots
+// const listContainer = document.querySelector('.list-container');
 const dots = new Image();
 dots.src = Dots;
 
-const tasks = [
-  { index: 0, description: 'Wash the dishes', completed: true },
-  { index: 1, description: 'Buy groceries', completed: true },
-  { index: 2, description: 'Do the laundry', completed: false },
-];
-
-const createTask = (tasks) => {
-  const taskContent = `
-  <li class="element">
-      <input type="checkbox" id="checkbox" class="checkbox" />
-      <div>${tasks.description}</div>
-      <div class="dots-container"><img class="dots"src=${Dots} /></div>
-  </li>`;
-
-  listContainer.innerHTML += taskContent;
-};
-
-const createList = (arr) => {
-  for (let i = 0; i < arr.length; i += 1) {
-    createTask(arr[i]);
+// task Class: represents a task object
+class Task {
+  constructor(id, description, completed) {
+    this.id = id;
+    this.description = description;
+    this.completed = completed;
   }
-};
+}
+// Store Class: Handles Storage
 
-createList(tasks);
+// Creating new Store instance
+const store = new Store();
+
+// Update Task
+function updateTask(e) {
+  const text = e.target.value;
+  const value = e.target.id;
+  const splitid = value.split('-');
+  const idstring = splitid[2];
+  const id = parseInt(idstring, 10);
+  const list = store.getList();
+  for (let i = 0; i < list.length; i += 1) {
+    const task = list[i];
+    if (task.id === id) {
+      list[i].description = text;
+    }
+  }
+  localStorage.setItem('list', JSON.stringify(list));
+}
+
+const inputTasks = [];
+// UI Class : Handles UI tasks
+class UI {
+  // Static so I don't have to instantiate
+
+  static displayList() {
+    const list = store.getList();
+    list.forEach((task) => UI.addTodoList(task));
+  }
+
+  static addTodoList(task) {
+    const tasksList = document.getElementById('list-container');
+
+    const taskContent = document.createElement('li');
+    taskContent.innerHTML = `
+        <input type="checkbox" id="checkbox" class="checkbox" />
+        <div><input value="${task.description}" class="task-input" id="task-input-${task.id}"/></div>
+        <div class="dots-container"><img class="dots"src="${Dots}" /></div>
+    `;
+    tasksList.appendChild(taskContent);
+    const taskInput = taskContent.querySelector('.task-input');
+    taskInput.addEventListener('keyup', updateTask);
+    inputTasks.push(taskInput);
+    taskContent.classList.add('element');
+    taskContent.setAttribute('id', `task-${task.id}`);
+  }
+
+  static deleteCompleted() {
+    const liList = document.querySelectorAll('.element');
+    liList.forEach((li) => {
+      const checkbox = li.querySelector('.checkbox');
```

Comment:
You have done great by implementing the function to delete all completed. Kindly implement a function to delete a particular item as required for this milestone. It also seems that you do not have a delete button. Kindly implement one and have it listen to the delete function.